samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -366,25 +296,20 @@ private async Task<string> GetChatResponseAsync(string chatId, SKContext chatCon
             chatContextText = $"{chatContextText}\n{chatHistory}";
         }
 
+        //Get the prompt.txt text
+        var parentDir = System.IO.Directory.GetCurrentDirectory() + "\\..\\..\\..";
+        var skillDir = Path.GetFullPath(Path.Combine(parentDir, "CopilotChat", "Skills", "SemanticSkills"));
```

Comment:
Abstract this: https://github.com/microsoft/semantic-kernel/blob/main/samples/dotnet/KernelHttpServer/Utils/RepoFiles.cs